docs/content/getting-started/faq.md

```diff
@@ -0,0 +1,114 @@
+# FAQ
+
+## Why is Traefik Answering XXX HTTP Response Status Code ?
+
+Traefik is a dynamic reverse proxy,
+and while the documentation often demonstrates configuration options through file examples,
+the core feature of Traefik is its ability to be provided with configuration dynamically.
+
+The file provider allows providing [dynamic configuration](../configuration-overview/#the-dynamic-configuration) through a file or directory.
+This is different from the [static configuration](../configuration-overview/#the-static-configuration),
+which can also be provided by a file.
+
+For HTTP configurations, the EntryPoint concept matches the Transport Layer (TCP),
+while the Router concept matches the Presentation (TLS) and Application layers (HTTP).
+
+An EntryPoint is not tied to the protocol HTTP or HTTPS,
+as one can attach a TLS router or a non-TLS router to the same EntryPoint.
```

Comment:
```suggestion
In other words, for a given Entrypoint, at any given time the traffic seen is not bound to be just about one protocol. It could be HTTP, or otherwise. Over TLS, or not. Not to mention that dynamic configuration changes potentially make that kind of traffic vary over time.
```